test/common/index.js

```diff
@@ -920,6 +925,7 @@ const common = {
   pwdCommand,
   requireNoPackageJSONAbove,
   runWithInvalidFD,
+  setHappyEyeballsEnabled,
```

Comment:
You'd need to add this to `test/common/index.mjs` as well.

Reply:
👍